backend/src/modules/tournament-tables/infrastructure/repository/mongo/MongoTournamentTablesRepository.ts

```diff
@@ -6,21 +6,20 @@ import { TableNumber } from '../../../core/domain/TableNumber';
 export class MongoTournamentTablesRepository implements TournamentTablesRepository {
   async save(tournamentTable: TournamentTable): Promise<void> {
     await MongoTournamentTables.findByIdAndUpdate(
```

Comment:
Tutaj powinno byc findOneAndUpdate.

To jak testy przechodza :) ?

Reply:
No właśnie nie przechodzą, właśnie wjechał nowy commit :)